packages/templates/src/home/Explorer.tsx

```diff
@@ -504,53 +505,109 @@ export const Template: VFC<
               />
 
               {currency && (
-                <Flex gap={3}>
-                  <InputGroup>
-                    <NumberInput
-                      clampValueOnBlur={false}
-                      min={0}
-                      step={Math.pow(10, -currency.decimals)}
-                      precision={currency.decimals}
-                      allowMouseWheel
-                      w="full"
-                      isDisabled={isSubmitting}
-                      onChange={(x: any) => setValue('minPrice', x)}
-                      format={(e) => e.toString()}
-                    >
-                      <NumberInputField
-                        placeholder={t('explore.form.min-price.placeholder')}
-                        {...register('minPrice')}
-                      />
-                      <NumberInputStepper>
-                        <NumberIncrementStepper />
-                        <NumberDecrementStepper />
-                      </NumberInputStepper>
-                    </NumberInput>
-                  </InputGroup>
-
-                  <InputGroup>
-                    <NumberInput
-                      clampValueOnBlur={false}
-                      min={0}
-                      step={Math.pow(10, -currency.decimals)}
-                      precision={currency.decimals}
-                      allowMouseWheel
-                      w="full"
-                      isDisabled={isSubmitting}
-                      onChange={(x: any) => setValue('maxPrice', x)}
-                      format={(e) => e.toString()}
-                    >
-                      <NumberInputField
-                        placeholder={t('explore.form.max-price.placeholder')}
-                        {...register('maxPrice')}
-                      />
-                      <NumberInputStepper>
-                        <NumberIncrementStepper />
-                        <NumberDecrementStepper />
-                      </NumberInputStepper>
-                    </NumberInput>
-                  </InputGroup>
-                </Flex>
+                <FormControl isInvalid={!!errors.minPrice || !!errors.maxPrice}>
+                  <Flex gap={3}>
+                    <InputGroup>
+                      <NumberInput
+                        clampValueOnBlur={false}
+                        step={Math.pow(10, -currency.decimals)}
+                        allowMouseWheel
+                        w="full"
+                        isDisabled={isSubmitting}
+                        onChange={(x: any) => setValue('minPrice', x)}
+                        min={0}
+                      >
+                        <NumberInputField
+                          id="minPrice"
+                          placeholder={t('explore.form.min-price.placeholder')}
+                          {...register('minPrice', {
+                            validate: (value) => {
+                              if (!value) return
```

Comment:
I think this should be removed because it bypass the other validations if the value is empty or zero.
Only this file have this logic, validation in other files do not have this.
Based on the typescript type, it should be:
```
if (value === null) return
```